Library/Homebrew/cmd/update.rb

```diff
@@ -41,8 +41,8 @@ def update
     master_updater.pull!
     master_updated = master_updater.updated?
     if master_updated
-      puts "Updated Homebrew from #{master_updater.initial_revision[0, 8]} " \
-           "to #{master_updater.current_revision[0, 8]}."
+      puts "Updated Homebrew from #{`git rev-parse --short #{master_updater.initial_revision}`.strip} " \
+           "to #{`git rev-parse --short #{master_updater.current_revision}`.strip}."
```

Comment:
I agree. Additionally, putting the abbreviated commits into separate local variables could improve the readability of this already too long `puts` invocation. I also believe `chomp` is sufficient (and is to be preferred over `strip`).